purchases/src/test/java/com/revenuecat/purchases/BasePurchasesTest.kt

```diff
@@ -154,6 +155,10 @@ internal open class BasePurchasesTest {
             } answers {
                 purchasesUpdatedListener = null
             }
+
+            every {
+                startConnectionOnMainThread()
+            } just runs
```

Comment:
Total nitpick, but I've been using `Runs` (uppercase) instead of `runs`. Do you have any preference?

Reply:
oh I have no preference, I will change it to the uppercase version 😄